src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/MessageIntegrationTest.kt

```diff
@@ -108,7 +107,7 @@ class MessageIntegrationTest : QueueIntegrationTest() {
       .expectBody()
       .jsonPath("index-status.currentIndex").isEqualTo(SyncIndex.INDEX_B.name)
       .jsonPath("index-status.inProgress").isEqualTo("true")
-      .jsonPath("index-size.${SyncIndex.INDEX_A.name}").isEqualTo("18")
+      .jsonPath("index-size.${SyncIndex.INDEX_A.name}").isEqualTo("20")
```

Comment:
dubious whether all these tests add value, don't they just make it a pain to change anything?  Maybe one should test the count, rest should check the value is at least...